atd-vze/src/containers/DefaultLayout/DefaultHeader.js

```diff
@@ -21,64 +22,84 @@ const propTypes = {
 
 const defaultProps = {};
 
+const EnvAlertBanner = () => {
+  const env = process.env.REACT_APP_HASURA_ENDPOINT.includes("staging")
+    ? "staging"
+    : process.env.NODE_ENV === "production"
+    ? "production"
+    : "local";
```

Comment:
is this nested ternary too gnarly? :sweat_smile: just using NODE_ENV shows production for netlify and isnt helpful for specifying local vs staging. using it in combo with REACT_APP_HASURA_ENDPOINT i could specify all the diff environments

Reply:
ya know, i kinda think we should go ahead and use a dedicated env variable for this like we do in Moped. i think we can just add `REACT_APP_ENV` to all of the `.env` files. it may come in handy again down the road anyway...

Reply:
I don't think the ternary is too gnarly at all. Not likely that we'll have more than these envs. I was originally going to say the staging check was alright, but John's review pushed my opinion over to updating the env files.

Also, making the alert yellow (`warning`) in the local env would be a cool touch to match up what we see in Moped Editor. That is not blocking this feature though - love the banner!

Reply:
@johnclary after your requested change is implemented, we may want to create a spin-off to tidy up the existing `isDev` var sprinkled around the app 🏗️

Reply:
oh, thanks for mentioning that. will do 👍 🙏